lib/wikitrivia_web/channels/game_channel.ex

```diff
@@ -13,4 +13,29 @@ defmodule WikitriviaWeb.GameChannel do
     broadcast! socket, "player_joined", %{player: player, players: players}
     {:noreply, socket}
   end
+
+  def handle_in("go", %{"game_id" => game_id}, socket) do
+    Game.start(game_id, game_timer_callback(game_id, socket))
+    {:noreply, socket}
+  end
+
+  def game_timer_callback(game_id, socket) do
+    fn ->
+      game_state = Game.get_game_state(game_id)
+      {message_type, message_data} = message_for_timer_state(game_state)
+      broadcast! socket, message_type, message_data
+    end
+  end
+
+  defp message_for_timer_state(%{timer_state: :question, timer_data: question_data, num_questions_left: num_questions_left}) do
+    {"start_question", %{question_number: 6 - num_questions_left, question: question_data}} # This question number calculation is dumb
```

Comment:
What do you think about always returning the complete representation of the game state to the client for each `broadcast!`? My rationale is that it will simplify the client side code, i.e. we won't have to modify the state on the client before redrawing the view.

Reply:
Yep that's fine with me, at the very least there's no reason to more narrowly optimize up front.